java/arcs/core/host/ArcState.kt

```diff
@@ -92,7 +91,7 @@ enum class ParticleState {
      * succeeded at least once).
      */
     val hasBeenStarted: Boolean
-        get() = listOf(FirstStart, Waiting, Started, Desynced, Stopped, Failed).contains(this)
+        get() = listOf(FirstStart, Waiting, Running, Desynced, Stopped, Failed).contains(this)
```

Comment:
Minor nit... might be nice to stash this list in a companion object val or something... instead of creating a new list every time `hasBeenStarted` is accessed

Reply:
I was assuming the compiler would optimise static lists like this... but I suppose that could be naive :)